src/protobuf/raft_serverpb.rs

```diff
@@ -507,13 +507,19 @@ impl ::protobuf::Clear for RaftMessage {
     }
 }
 
+impl ::std::fmt::Debug for RaftMessage {
```

Comment:
Which controls the codegen behavior? Can we ensure everyone generates the right `Debug` implementation?

Reply:
I remove the check in codegen logic. https://github.com/BusyJay/rust-protobuf/commit/7da1412e210cfc50e881159428c4cf708653d2ed#diff-6499709ed714601b45000c51bfa00217